definitions.h

```diff
@@ -24,6 +24,7 @@
 #define MAX3(A,B,C) MAX((MAX(A,B)),C)
 #define X_KEY 0
 #define Y_KEY 1
+#define DEG_TO_RAD M_PI / 180
```

Comment:
Good use for this. I might have to add something like that in mine. It looks cleaner.